app/viewmodels/govuk.scala

```diff
@@ -16,19 +16,17 @@
 
 package viewmodels
 
-package object govuk {
-
-  object GovukFluency
-    extends BackLinkFluency
-      with ButtonFluency
-      with CheckboxFluency
-      with DateFluency
-      with ErrorSummaryFluency
-      with FieldsetFluency
-      with HintFluency
-      with InputFluency
-      with LabelFluency
-      with PhaseBannerFluency
-      with RadiosFluency
-      with TagFluency
-}
+package object govuk
+  extends ImplicitConversions
+  with govuk.BackLinkFluency
+  with govuk.ButtonFluency
+  with govuk.CheckboxFluency
+  with govuk.DateFluency
+  with govuk.ErrorSummaryFluency
+  with govuk.FieldsetFluency
+  with govuk.HintFluency
+  with govuk.InputFluency
+  with govuk.LabelFluency
+  with govuk.PhaseBannerFluency
+  with govuk.RadiosFluency
+  with govuk.SummaryListFluency
```

Comment:
This may be a bit too much - we could also consider:
```suggestion
package object govuk {
  object all extends ImplicitConversions
    with govuk.BackLinkFluency
    with govuk.ButtonFluency
    with govuk.CheckboxFluency
    with govuk.DateFluency
    with govuk.ErrorSummaryFluency
    with govuk.FieldsetFluency
    with govuk.HintFluency
    with govuk.InputFluency
    with govuk.LabelFluency
    with govuk.PhaseBannerFluency
    with govuk.RadiosFluency
    with govuk.SummaryListFluency
}
```
so that all of this is not brought into scope accidentally by IDEs changing multiple imports into a wildcard one